apps/docs/components/nuxt.config.ts

```diff
@@ -13,6 +14,12 @@ export default defineNuxtConfig({
       ],
     },
   },
+  robots: {
+    enabled: false
```

Comment:
why do we disable robots on docs? dont we wanna to be docs visible in search engines? or am I missing something